guides/logging.txt

```diff
@@ -0,0 +1,24 @@
+TOPIC
+    logging
+
+DESCRIPTION
+    Bolt logs messages both to the console and to log files. Messages can
+    either come from Bolt's internal 'outputter' class, which logs user-facing
+    messages like progress and results, or from the 'logger' class, which logs
+    warnings, errors, and log-structured output to log files. Both of these message
+    streams are configurable.
+
+    By default, Bolt logs to the console at 'warn' level, writes a log file to
+    '<project>/bolt-debug.log' at 'debug' level, and runs with the outputter in
+    verbose mode unless you are running a plan, in which case it runs without
+    verbose (you still with me?). You can configure logging output in your
+    project configuration, and can configure console log levels (but not file log
+    levels) with `--log-level`. For example, we recommend running `--log-level
+    debug` to quickly debug things like connection issues and unexpected run
+    results. You can configure the user-facing output only on the command line with
+    `--[no]-verbose`.
```

Comment:
```suggestion
    verbose (you still with me?). You can configure logging output in your
    project configuration, and can configure console output from the command
    line.
```

As with the other guides, I think we should avoid documenting specific commands, config options, etc., as that's more documentation. It also gets a little weird showing command-line options, as they're not the same between *nix shells and PowerShell.

Reply:
I'm starting to wonder if all of this is really just documentation - this is mainly to address Dimitri's question of whether `verbose` is a log level or not, but...there's not much to say that's *not* explanatory besides "Verbose is different from logging".

Reply:
Maybe we could have this *just* link to the docs? We don't do that with other guides, but this could be the first

Reply:
After reading this, I do think the logging doc needs some updates. I don't think it hurts to have a guide though 🤷